producer/producer_sf.go

```diff
@@ -80,6 +80,9 @@ func ParseSampledHeader(flowMessage *flowmessage.FlowMessage, sampledHeader *sfl
 				tostmp := uint32(binary.BigEndian.Uint16(data[offset : offset+2]))
 				tos = uint8(tostmp & 0x0ff0 >> 4)
 				ttl = data[offset+7]
+
+				flowLabeltmp := binary.BigEndian.Uint32(data[offset : offset+4])
+				(*flowMessage).SrcPort = flowLabeltmp & 0x3FF
```

Comment:
I think this should read
```suggestion
				(*flowMessage).IPv6FlowLabel = flowLabeltmp & 0x000fffff
```